app/(pages)/about-us/page.tsx

```diff
@@ -1,10 +1,89 @@
-import { Stack } from '@mui/joy';
+import { Stack, Box, Typography, Card, CardOverflow, AspectRatio, CardContent,Sheet } from '@mui/joy';
+import Image from 'next/image';
 import PageHeaderCard from "@/components/PageHeaderCard";
 
+
+interface cardProps {
+  picture: string;
+  cardName: string;
+  cardText: string;
+}
+
+function AboutUsCard({picture, cardName, cardText}: cardProps) {
+  return (
+      <Card className='light-wight' variant='plain' sx={{ width:'585px', height:'566px', boxShadow: 3 }}>
+        <CardOverflow>
+          <AspectRatio ratio='2'>
+            <Image 
+            src={picture}
+            alt=''
+            />
+          </AspectRatio>
+        </CardOverflow>
+
+        <CardContent>
+          <Stack direction='column' alignItems='center'>
+            <Typography fontSize='30px' pb={3} pt={2} fontWeight='530'
+              sx={{textAlign:'center'}}>
+              {cardName}
+            </Typography>
+            <Typography fontSize='16px' px={2} fontWeight='250'
+              sx={{textAlign:'center'}}>
+              {cardText}
+            </Typography>
+          </Stack>
+        </CardContent>
+      </Card>
+  );
+
+}
 export default function AboutUs() {
+
+  const aboutWIEsocText = 'We are a UNSW student-run society that facilitates members from all \n \
+                          disciplines of engineering. We run many social and industry events \n \
+                          throughout the year as well as two programs dedicated to mentoring \n \
+                          and interpersonal development. Our society provides a supportive network \n \
+                          for all our members, whether past, present or future students, to advocate \n \
+                          gender acceptance and equality in all facets of the engineering industry.'
+
+
+            
   return (
+    <Sheet>
     <Stack>
       <PageHeaderCard imagePath={'/about-us/header.jpg'} pageTitle={'About Us'}/>
     </Stack>
+      <center>
```

Comment:
this is a deprecated feature 
<img width="714" alt="image" src="https://github.com/UNSW-WIESoc/website/assets/119286195/72fbaa92-f064-4c90-9f77-e03b6467889e">
pls use flex styling on the Stack above it instead, e.g. justifyContent, alignItems